packages/components/psammead-media-indicator/src/index.stories.jsx

```diff
@@ -48,11 +76,14 @@ storiesOf('Components|MediaIndicator', module)
     { notes },
   )
   .add(
-    'audio without duration',
+    'top story audio with duration',
     () => (
       <MediaIndicator
         offscreenText={text('offscreenText', 'Audio')}
+        duration={text('duration', '2:15')}
+        datetime={text('datetime', 'PT2M15S')}
         type="audio"
+        topStory="true"
```

Comment:
```suggestion
        topStory
```
`topStory` is a boolean not a string

Reply:
Of course 🤦🏻‍♂️